osu.Game/Skinning/SkinInfo.cs

```diff
@@ -39,7 +39,10 @@ public class SkinInfo : RealmObject, IHasRealmFiles, IEquatable<SkinInfo>, IHasG
 
         public bool Protected { get; set; }
 
-        public virtual Skin CreateInstance(IStorageResourceProvider resources)
+        /// <summary>
+        /// Returns the specific subtype of <see cref="Skin"/> that will be constructed on calling <see cref="CreateInstance"/>.
```

Comment:
Except it returned `TrianglesSkin` in a case where it's no longer returned in `CreateInstance`? This change makes zero sense when only looking at the diff, so I'm going to require further explanation.

Reply:
I don't understand your concern. This is just splitting what was in `CreateInstance` to its own method, I'm not sure what's wrong here.